AudioComments/AudioComments/AudioRecorderController.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  AudioRecorderController.swift
+//  AudioComments
+//
+//  Created by Alex Thompson on 5/12/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import UIKit
+import AVFoundation
+
+class AudioRecorderController: UIViewController {
+    
+    var audioPlayer: AVAudioPlayer? {
+        didSet {
+            guard let audioPlayer = audioPlayer else { return }
+            
+//            audioPlayer.delegate = self
```

Comment:
Remove commented code prior to submitting.

Reply:
Will do, its commented out because i planned on using it but hadnt added the delegate extension yet